dotnet/src/SemanticKernel/CoreSkills/TextMemorySkill.cs

```diff
@@ -46,6 +46,19 @@ public class TextMemorySkill
     private const string DefaultRelevance = "0.75";
```

Comment:
I like the constructor change here. I think with this, we should make DefaultRelevance = "0.0", 0.75 is very ADA specific